Renamed `getHeaders()` to `getHeadings()` (#1107)
withastro/docs: src/pages/en/guides/markdown-content.md

If it's a page, URL of the page (e.g. `/en/guides/markdown-content`).

#### `getHeadings()`

An async function that returns the headings of the Markdown file. This function returns the type: `{ depth: number; slug: string; text: string }[]`.

#### `getHeaders()`

:::caution[Deprecated]

> The `getHeaders()` function has been renamed to `getHeadings()`. Please update your code to use `getHeadings()`.

Reviewer: ```suggestion
The `getHeaders()` function has been renamed. To update an older project, please use `getHeadings()` instead.
```
Just a suggestion to be more explicit about which one to use. Similarly, a note about this should end up in the migration guide. (You could probably use this as-is there.)
